Source/Plugins/EditorModules/ObjectInspector/ObjectInspectorSettings.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Xml.Linq;
+
+namespace Duality.Editor.Plugins.ObjectInspector
+{
+	public class ObjectInspectorSettings
+	{
+		public List<ObjectInspectorState> ObjectInspectors { get; set; } = new List<ObjectInspectorState>();
+	}
+
+	public class ObjectInspectorState
+	{
+		private int id;
```

Comment:
Style: Please group all the fields and all the properties separately, as seen [here](https://github.com/Barsonax/duality/blob/ba2f86696a6dd28d92fb2d7b9061d1938091e96c/Source/Plugins/EditorModules/LogView/LogViewSettings.cs).